06/02/2022 02:36:32 PM : ERROR : devf : active_logger : 38 : (Process Details : (53240, MainProcess), Thread Details : (63720, ScriptRunner.scriptThread))
Log Message : Error Type : KeyError, Error Message : 'formatters'
06/02/2022 02:47:33 PM : ERROR : devf : active_logger : 38 : (Process Details : (75328, MainProcess), Thread Details : (104844, ScriptRunner.scriptThread))
Log Message : Error Type : KeyError, Error Message : 'formatters'
06/02/2022 02:56:36 PM : ERROR : devf : active_logger : 38 : (Process Details : (74836, MainProcess), Thread Details : (25780, ScriptRunner.scriptThread))
Log Message : Error Type : KeyError, Error Message : 'formatters'
02-06-2022 03:02:55 : ERROR : devf : get_yfdata : 71 : (Process Details : (82404, MainProcess), Thread Details : (18564, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 03:06:29 : ERROR : devf : get_yfdata : 71 : (Process Details : (82404, MainProcess), Thread Details : (98460, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 03:06:29 : ERROR : app : main : 57 : (Process Details : (82404, MainProcess), Thread Details : (98460, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 03:06:45 : ERROR : devf : get_yfdata : 71 : (Process Details : (82404, MainProcess), Thread Details : (39020, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 03:06:45 : ERROR : app : main : 57 : (Process Details : (82404, MainProcess), Thread Details : (39020, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 03:10:43 : ERROR : devf : active_logger : 39 : (Process Details : (82404, MainProcess), Thread Details : (32784, ScriptRunner.scriptThread))\nLog : Error Type : DuplicateSectionError, Error Message : While reading from 'log.conf' [line 32]: section 'handler_file' already exists
06/02/2022 03:10:43 PM : ERROR : devf : active_logger : 39 : (Process Details : (82404, MainProcess), Thread Details : (32784, ScriptRunner.scriptThread))
Log Message : Error Type : DuplicateSectionError, Error Message : While reading from 'log.conf' [line 32]: section 'handler_file' already exists
02-06-2022 03:10:44 : ERROR : devf : get_yfdata : 72 : (Process Details : (82404, MainProcess), Thread Details : (32784, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 03:10:44 : ERROR : app : main : 57 : (Process Details : (82404, MainProcess), Thread Details : (32784, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
06/02/2022 03:12:26 PM : ERROR : devf : active_logger : 39 : (Process Details : (21300, MainProcess), Thread Details : (8600, ScriptRunner.scriptThread))
Log Message : Error Type : DuplicateSectionError, Error Message : While reading from 'log.conf' [line 32]: section 'handler_file' already exists
02-06-2022 03:15:29 : ERROR : devf : get_yfdata : 72 : (Process Details : (42728, MainProcess), Thread Details : (33584, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 03:15:29 : ERROR : app : main : 57 : (Process Details : (42728, MainProcess), Thread Details : (33584, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 03:16:17 : ERROR : devf : get_yfdata : 72 : (Process Details : (42728, MainProcess), Thread Details : (88304, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 03:16:17 : ERROR : app : main : 57 : (Process Details : (42728, MainProcess), Thread Details : (88304, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 03:32:49 : ERROR : app : main : 58 : (Process Details : (95588, MainProcess), Thread Details : (61364, ScriptRunner.scriptThread))\nLog : Error Type : TypeError, Error Message : 'module' object is not callable
02-06-2022 03:34:48 : ERROR : app : main : 60 : (Process Details : (95588, MainProcess), Thread Details : (28072, ScriptRunner.scriptThread))\nLog : Error Type : TypeError, Error Message : 'module' object is not callable
02-06-2022 03:59:57 : ERROR : app : main : 60 : (Process Details : (8124, MainProcess), Thread Details : (13252, ScriptRunner.scriptThread))\nLog : Error Type : TypeError, Error Message : 'module' object is not callable
02-06-2022 04:02:13 : ERROR : devf : get_yfdata : 71 : (Process Details : (14156, MainProcess), Thread Details : (15824, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : "None of ['Date'] are in the columns"
02-06-2022 04:02:13 : ERROR : app : main : 60 : (Process Details : (14156, MainProcess), Thread Details : (15824, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 04:02:47 : ERROR : devf : get_yfdata : 71 : (Process Details : (14156, MainProcess), Thread Details : (13064, ScriptRunner.scriptThread))\nLog : Error Type : TypeError, Error Message : DataFrame.set_index() missing 1 required positional argument: 'keys'
02-06-2022 04:02:47 : ERROR : app : main : 60 : (Process Details : (14156, MainProcess), Thread Details : (13064, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 04:05:59 : ERROR : devf : get_yfdata : 71 : (Process Details : (10128, MainProcess), Thread Details : (14512, MainThread))\nLog : Error Type : TypeError, Error Message : DataFrame.set_index() missing 1 required positional argument: 'keys'
02-06-2022 04:05:59 : ERROR : app : main : 60 : (Process Details : (10128, MainProcess), Thread Details : (14512, MainThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 04:07:35 : ERROR : devf : get_yfdata : 73 : (Process Details : (8456, MainProcess), Thread Details : (7176, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : 'Date'
02-06-2022 04:07:35 : ERROR : app : main : 60 : (Process Details : (8456, MainProcess), Thread Details : (7176, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 04:07:57 : ERROR : devf : get_yfdata : 73 : (Process Details : (8456, MainProcess), Thread Details : (17492, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : 'Date'
02-06-2022 04:07:57 : ERROR : app : main : 60 : (Process Details : (8456, MainProcess), Thread Details : (17492, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 04:17:04 : ERROR : header : add : 62 : (Process Details : (8456, MainProcess), Thread Details : (14964, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'int' object has no attribute 'style'
02-06-2022 04:17:24 : ERROR : header : add : 61 : (Process Details : (8456, MainProcess), Thread Details : (7932, ScriptRunner.scriptThread))\nLog : Error Type : NameError, Error Message : name 'value' is not defined
02-06-2022 04:25:26 : ERROR : devf : get_yfdata : 73 : (Process Details : (8456, MainProcess), Thread Details : (2544, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : 'Datetime'
02-06-2022 04:25:26 : ERROR : app : main : 60 : (Process Details : (8456, MainProcess), Thread Details : (2544, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
02-06-2022 04:25:46 : ERROR : devf : get_yfdata : 73 : (Process Details : (8456, MainProcess), Thread Details : (4036, ScriptRunner.scriptThread))\nLog : Error Type : KeyError, Error Message : 'Datetime'
02-06-2022 04:25:46 : ERROR : app : main : 60 : (Process Details : (8456, MainProcess), Thread Details : (4036, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
03-06-2022 10:02:50 : ERROR : dashboard : tableau : 34 : (Process Details : (4084, MainProcess), Thread Details : (16772, ScriptRunner.scriptThread))\nLog : Error Type : ValueError, Error Message : yellow is not valid. Available options: ['streamlit', 'light', 'dark', 'blue', 'fresh', 'material']
03-06-2022 10:03:38 : ERROR : dashboard : tableau : 34 : (Process Details : (4084, MainProcess), Thread Details : (21688, ScriptRunner.scriptThread))\nLog : Error Type : ValueError, Error Message : ag-theme-alpine-dark is not valid. Available options: ['streamlit', 'light', 'dark', 'blue', 'fresh', 'material']
03-06-2022 11:05:51 : ERROR : devf : get_yfdata : 76 : (Process Details : (4084, MainProcess), Thread Details : (16400, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'DataFrame' object has no attribute 'map'
03-06-2022 11:05:51 : ERROR : app : main : 61 : (Process Details : (4084, MainProcess), Thread Details : (16400, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
03-06-2022 11:06:03 : ERROR : app : main : 61 : (Process Details : (4084, MainProcess), Thread Details : (17596, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
03-06-2022 11:10:48 : ERROR : devf : get_yfdata : 77 : (Process Details : (4084, MainProcess), Thread Details : (14516, ScriptRunner.scriptThread))\nLog : Error Type : ValueError, Error Message : Invalid format specifier
03-06-2022 11:10:48 : ERROR : app : main : 61 : (Process Details : (4084, MainProcess), Thread Details : (14516, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
03-06-2022 11:11:12 : ERROR : app : main : 61 : (Process Details : (4084, MainProcess), Thread Details : (15976, ScriptRunner.scriptThread))\nLog : Error Type : AttributeError, Error Message : 'NoneType' object has no attribute 'empty'
